allure-report-face/src/styles.css

```diff
@@ -10,11 +11,13 @@
 @import "./blocks/link/styles.css";
 @import "./blocks/spinner/styles.css";
 @import "./blocks/text/styles.css";
+@import "./blocks/widget/styles.css";
 @import "./variables.css";
+@import url(https://fonts.googleapis.com/css?family=Open+Sans:400,300,700&subset=latin,cyrillic);
```

Comment:
I am using head.

1. It increases size of allure artifiact by 70Kb. Every report will be generated and transferred slowly because of fancy font face.
2. Maybe it looks better on your system setup in your environment. But people are different. Some of them would like to use upscaled system fonts, some of them even on Ubuntu. So, that's why I'd like to use something that already exists in system (and therefore optimised).